.jenkins/pytorch/codegen-test.sh

```diff
@@ -37,7 +37,6 @@ python -m tools.setup_helpers.generate_code \
 mkdir -p "$OUT"/pyi/torch/_C
 mkdir -p "$OUT"/pyi/torch/nn
 python -m tools.pyi.gen_pyi \
-  --declarations-path "$OUT"/torch/share/ATen/Declarations.yaml \
```

Comment:
I should have taken this out in the original PR

Reply:
I should have integrated this script with CI properly...